docs/podman-images.1.md

```diff
@@ -120,6 +123,14 @@ REPOSITORY   TAG      IMAGE ID       CREATED       SIZE
 ]
 ```
 
+```
+# podman images --sort repository
+REPOSITORY                                   TAG      IMAGE ID       CREATED       SIZE
+<none>                                       <none>   ebb91b73692b   4 weeks ago   27.2MB
+docker.io/kubernetes/pause                   latest   e3d42bcaf643   3 years ago   251kB
```

Comment:
Nit could you give me an example with some containers other then docker.io add in localhost and quay.io.